### PR TITLE
Sle and leap support

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

**Comment by imobachgs** [comment]
LGTM, thanks!

**Comment by lslezak** [comment]
LGTM, thanks!

File: data/targets.yml [hunk 1]
```diff
@@ -13,6 +23,16 @@
   obs_project: "Devel:YaST:Head"
   obs_sr_project: "SUSE:SLE-12-SP2:GA"
   obs_target: "SLE-12-SP1"
+:sle12sp3:
+  obs_api: "https://api.suse.de/"
+  obs_project: "Devel:YaST:Head"
+  obs_sr_project: "SUSE:SLE-12-SP3:GA"
+  obs_target: "sle12_sp3"
+:sle_latest:
```
Comment (lslezak) [inline data/targets.yml]: NP: I was just thinking about a reference in the YAML to share the same part:

```yaml
:sle12sp3: &1
  obs_api: "https://api.suse.de/"
  ...
:sle_latest: *1
```

But that would be probably less readable...

Comment (jreidinger) [inline data/targets.yml]: yeap, I know it, but I try to be KISS,  I know it is not DRY, but it is obvious where we send what.

**Pull request merged**